lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -381,6 +393,88 @@ def stop_event_loop(self):
     stop_event_loop.__doc__=FigureCanvasBase.stop_event_loop_default.__doc__
 
 
+_flow = [Gtk.Orientation.HORIZONTAL, Gtk.Orientation.VERTICAL]
+flow_types = ['horizontal', 'vertical']
+
+
+class Window(WindowBase, Gtk.Window):
```

Comment:
I kind of agree with you, but in that case, why don't we remove all the `xxxGTK3` stuff? and leave just `xxx` like `FigureManager`, `Toolbar`, etc...